src/elements/content-explorer/ItemGridCell.js

```diff
@@ -24,10 +26,20 @@ const ItemGridCell = ({
     view,
     ...rest
 }: Props) => {
+    const url = getProp(item, `thumbnailUrl`);
+    const itemClassName = classNames(
+        'bce-ItemGridCell-item',
+        { 'bce-ItemGridCell-item--postLoadThumbnail': url },
+        { 'bce-ItemGridCell-item--itemIcon': !url },
+    );
     return (
-        <figure>
+        <figure className="bce-itemGridCell-figure">
             <div className="bce-ItemGridCell-itemThumbnail">
-                <div className="bce-ItemGridCell-itemIcon">{getIcon(128, item)}</div>
+                {url ? (
+                    <div className={itemClassName} style={{ backgroundImage: `url("${url}")` }} />
+                ) : (
+                    <div className={itemClassName}> {getIcon(128, item)} </div>
```

Comment:
Please avoid single non-breaking spaces in markup, as they can change the way it's rendered in some browsers.